src/frontend/hooks/goose-form.js

```diff
@@ -144,10 +144,9 @@ const useGooseForm = () => {
           }
         } 
         else {
-          // // TODO when new goose is implemented
           const res = await api.geeseInfo.addGeeseInfo(gooseInfo);
           await api.geeseInfo.addGeeseImages(
-            imageUrls2.map((image) => {
+            imageUrls.map((image) => {
```

Comment:
This line causes the bug of not being able to save the goose form for new gooses. This is due to using `imageUrls`, which is a map of the previous images, which in a new goose, is empty regardless of the images you add. I believe `newlyAddedImageUrls` should be used instead, which represents the new images. However, all in all, this naming convention of the `images` state representing new images and `imageUrls` representing previous images isn't great